### PR TITLE
Disable collections without geometries in buildings/TIGER selects

### LINKED ISSUE #321: Check for non-null geometries for TIGER and buildings assets
Similar to #279

### PR DESCRIPTION
**Pull request by janakdr**
Factor out logic used for damage and use it for the state selects as well, for state assets that should have geometries (TIGER and buildings).

Closes #321.

### CODE REVIEW

**Comment by janakdr** [comment]
Reminder to myself to change base branch back to master

File: docs/import/list_ee_assets.js [hunk 1]
```diff
@@ -24,14 +24,14 @@ function getStateAssetsFromEe(state) {
     return maybeStatePromise;
   }
```
Comment (juliexxia) [inline docs/import/list_ee_assets.js]: add hasGeometry to return type?

File: docs/import/manage_disaster.js [hunk 2]
```diff
@@ -515,12 +517,21 @@ function initializeScoreSelectors(states, stateAssets) {
   }
 
   // For each asset type, add select for all assets for each state.
-  for (const {idStem, propertyPath, expectedColumns} of scoreAssetTypes) {
+  for (const {idStem,
+              propertyPath,
```
Comment (juliexxia) [inline docs/import/manage_disaster.js]: also add hasGeometry to this sig? (and others?)

File: cypress/integration/unit_tests/manage_disaster_set_score_parameters_test.js [hunk 3]
```diff
@@ -26,6 +26,7 @@ describe('Score parameters-related tests for manage_disaster.js', () => {
   before(preparePage);
 
   let stateStub;
+  let disasterStub;
```
[the remark this answers is not in the document]
Comment (janakdr) [reply cypress/integration/unit_tests/manage_disaster_set_score_parameters_test.js]: Same issue as other PR

File: docs/import/list_ee_assets.js (hunk 1 above)
Comment (janakdr) [reply docs/import/list_ee_assets.js]: Done.

File: docs/import/manage_disaster.js (hunk 2 above)
Comment (janakdr) [reply docs/import/manage_disaster.js]: Think I got all of them!

File: docs/import/list_ee_assets.js [hunk 4]
```diff
@@ -52,58 +54,74 @@ const disasterAssetPromises = new Map();
...
-          .then((disableList) => {
-            const assetMap = new Map();
-            const disableListIterator = disableList[Symbol.iterator]();
-            for (const {asset, type} of listEeAssetsResult) {
-              assetMap.set(
-                  asset, {type, disabled: disableListIterator.next().value});
+      markHasGeometryAssets(listEeAssets(eeLegacyPathPrefix + disaster))
+          .then((assetMap) => {
+            for (const attributes of assetMap.values()) {
+              attributes.disabled = !attributes.hasGeometry;
```
Comment (janakdr) [inline docs/import/list_ee_assets.js]: whoops: this disables all non-FCs, which is not what we should be doing :/

**Pull request merged**